idanalysis/analysis.py

```diff
@@ -1285,23 +1332,28 @@ def plot_optics_corr_results(
             _plt.show()
         return True
 
-    def run_correction_algorithms(self):
+    def run_correction_algorithms(self, nom_model=None):
         """Run correction algorithms.
 
+        Args:
+            nom_model (pymodels ob._, optional): Lattice. Defaults to None.
+
         Raises:
             ValueError: Error if calc type is invalid.
 
         Returns:
-            Bool: True
+            _Bool: True
```

Comment:
did you mean this underscore char in the beginning of `Bool` ? If the method only returns `True` an eceptions I think it is more appropriate not to return anything,